docs/license.rst

```diff
@@ -0,0 +1,44 @@
+About the *NiPreps* framework licensing
+---------------------------------------
+Please check https://www.nipreps.org/community/licensing/ for detailed
+information on the criteria we use to license *fMRIPrep* and other
+projects of the framework.
+
+License information
+-------------------
+Copyright (c) 2023, the *NiPreps* Developers.
+
+As of the 21.0.x pre-release and release series, *fMRIPrep* is
+licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Copyright (c) 2015-2023, the *fMRIPrep* developers and the CRN.
+All rights reserved.
+
+*fMRIPrep* 20.2 series and earlier are
+licensed under the BSD 3-clause license.
+You may obtain a copy of the License at
+https://opensource.org/licenses/BSD-3-Clause
+
+All trademarks referenced herein are property of their respective holders.
+
+The ``fmriprep-wrapper`` for Docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Copyright (c) 2020-2023, the *NiPreps* Developers.
+Copyright (c) 2015-2020, the *fMRIPrep* developers and the CRN.
+All rights reserved.
+
+*fMRIPrep-wrapper* is
+licensed under the BSD 3-clause license.
+You may obtain a copy of the License at
+https://opensource.org/licenses/BSD-3-Clause
+
+All trademarks referenced herein are property of their respective holders.
```

Comment:
Are you planning on packaging a wrapper?

```suggestion
```

Reply:
Good point. I think just a Docker image should be enough.